packages/service/src/utils/users.ts

```diff
@@ -1,5 +1,26 @@
+import { upsert } from "zapatos/db";
 import { db, pool, schema } from "../db";
 
 export function createUser(props: { name: string }) {
-  return db.upsert("users", props, "name").run(pool);
+  // return db.upsert("users", props, "name").run(pool);
+  return db.sql<
+    schema.users.SQL,
+    schema.users.Selectable
+  >`INSERT INTO ${"users"} (${"name"}) VALUES (${{ name: props.name }})`.run(
```

Comment:
We should add the password into this statement. Could we use the insert helpers here?

It would be cool if something like this worked:
```ts
db.insert('users', {
  name,
  password: db.sql`crypt(${db.param(password)}, gen_salt('md5'))`
})
```